### ISSUE TITLE
Meshes get erased while loading Collada model

### ISSUE THREAD

**Issue by idilsulo** [mikedh/trimesh#1853]
Hi!

I am using Trimesh to import models that are in Collada (DAE) format with their texture, in order to export them to OBJ + MTL files. However, for some of my models, I figured out that certain meshes are being erased right after they are loaded from Collada format.

The issue can be replicated by:

```
trimesh = trimesh.load('keyboard/model.dae')
trimesh.view()
```

The above lines of code results in the following view of the model:

![trimesh](https://user-images.githubusercontent.com/19615018/220760710-a84783d6-41f7-401f-96c8-0a4ffd4aa5ac.png)

Whereas the actual `model.dae` file is displayed as following on MacOS finder:

<img width="578" alt="dae" src="https://user-images.githubusercontent.com/19615018/220760968-65632aba-964a-4596-868a-d2584b09b607.png">

Basically for the case of the Trimesh object, some meshes are deleted, i.e. keys of the keyboard are lost.

Is there a workaround for this?

Thanks a lot!

**Comment by mikedh** [comment]
Oh interesting, would you be able to share the model this is happening on?

**Comment by idilsulo** [comment]
Sure, here you can find the model and its corresponding textures:
[keyboard.zip](https://github.com/mikedh/trimesh/files/10809911/keyboard.zip)

**Comment by mikedh** [comment]
Thanks for the model! It looks like a name collision bug, which is fixed by using `util.unique_name` which should release in #1844:

![Screenshot from 2023-02-22 22-19-46](https://user-images.githubusercontent.com/1403074/220814509-40e7b18e-2619-4fde-a6c8-df9c0963b5ff.png)
